cmd/probe.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"github.com/kubearmor/kubearmor-client/probe"
+	"github.com/spf13/cobra"
+)
+
+var probeInstallOptions probe.ProbeOptions
+
+// probeCmd represents the get command
+var probeCmd = &cobra.Command{
+	Use:   "probe",
+	Short: "Display probe information",
+	Long:  `Display probe information`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := probe.PrintProbeResult(client, probeInstallOptions); err != nil {
+			return err
+		}
+		return nil
+		
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(probeCmd)
+	probeCmd.Flags().StringVarP(&probeInstallOptions.Namespace, "namespace", "n", "default", "Namespace for resources")
+	probeCmd.Flags().StringVarP(&probeInstallOptions.ProbeDaemonImage, "image", "i", "alpine", "Kubearmor daemonset image to use")
```

Comment:
```suggestion
	probeCmd.Flags().StringVarP(&probeInstallOptions.ProbeDaemonImage, "image", "i", "alpine", "Probe Daemonset Image to use")
```

What would be the usecase here for providing custom image?

Reply:
No use case actually, forgot to take that out